flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/util/GenericHiveMetastoreCatalogUtil.java

```diff
@@ -118,18 +123,26 @@ public static Table createHiveTable(ObjectPath tablePath, CatalogBaseTable table
 				hiveTable.setPartitionKeys(new ArrayList<>());
 			}
 
-			hiveTable.setSd(sd);
```

Comment:
I'm wondering why we remove this line. If this is not needed, then we should probably remove everything about Sd.